vaadin-icons-flow-parent/vaadin-icons-flow/src/test/java/com/vaadin/flow/component/icon/tests/FontIconTest.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2000-2023 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.icon.tests;
+
+import com.vaadin.flow.component.icon.FontIcon;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class FontIconTest {
+
+    @Test
+    public void constructor_hasFont() {
+        var icon = new FontIcon("fa-solid", "fa-user");
```

Comment:
Would it be worth to create a IT test with font awesome integration to ensure that the whole chain with flow -> web components -> flow components work? Testbench could even be used to create and compare screenshots later on to ensure that not the DOM but also the visual appearance is correct.

Reply:
I did verify this manually in a test application but an IT would be a good addition here as well.

Reply:
Note that such ITs can't be added before the Flow Components are using the next alpha version of the web components (I needed to manually hack the test app to use the updated vaadin-icon).

Reply:
Seems that the current alpha already had the `font` API for icon available, so I was able to add the IT